src/Tutorials/Basics/hda_ideal_VLE.py

```diff
@@ -73,36 +76,39 @@ def build(self):
                            "Vap": self.component_list}
 
         # List of phase equilibrium index
-        self.phase_equilibrium_idx = Set(initialize=[1, 2, 3, 4, 5])
+        self.phase_equilibrium_idx = Set(initialize=[1, 2, 3, 4])  #, 5])
```

Comment:
Commented index here and lines below. These need to be cleaned up. As mentioned before, why are we even adding diphenyl in these (we deliberately left it out for simplicity in the original example).